integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonInsertIntoHadoopFsRelationCommand.scala

```diff
@@ -199,7 +199,7 @@ case class CarbonInsertIntoHadoopFsRelationCommand(
       // refresh cached files in FileIndex
       fileIndex.foreach(_.refresh())
       // refresh data cache if table is cached
-      sparkSession.catalog.refreshByPath(outputPath.toString)
+      // sparkSession.catalog.refreshByPath(outputPath.toString)
```

Comment:
1. why commented it?
2. if no need already, remove it directly.

Reply:
done